### ISSUE TITLE
several warnings in ansible 2.0.x

### ISSUE THREAD

**Issue by SydOps** [devops-coop/ansible-haproxy#41]
1. `tasks/install.yml`

```
[DEPRECATION WARNING]: Instead of sudo/sudo_user, use become/become_user and make sure become_method is 'sudo' (default).
This feature will be removed in a future release. Deprecation
 warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```
2. 
```
TASK [haproxy : Check epel repo] ***********************************************
 [WARNING]: Consider using yum module rather than running yum
```
3. 
```
TASK [haproxy : Build up the listen sections] **********************************
[DEPRECATION WARNING]: Using bare variables is deprecated. Update your playbooks so that the environment value uses the full variable syntax ('{{haproxy_listen}}').
This feature will
be removed in a future release. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

**Comment by SydOps** [comment]
just warning, not problem, closed it.

**Comment by PierrePaul** [comment]
Thanks for the report, I'm reopening, we should fix this.

**Comment by PierrePaul** [comment]
I I will have to to aalumin morning I'M aluminium and yu uy aluminium this
I'M ON to I I I will be removal tool to help tyi

On Oct 29, 2016 7:04 i bii or mute the thread.
I I I I I I uuuy
It It

**Comment by PierrePaul** [comment]
I I will have to to aalumin morning I'M aluminium and yu uy aluminium this
I'M ON to I I I will be removal tool to help tyi

On Oct 29, 2016 7:04 i bii or mute the thread.
I I I I I I uuuy
It It

On Oct 29, 2016 7:04 PM, "Ben Webber" <notifications@github.com> wrote:

Closed #41 <https://github.com/devops-coop/ansible-haproxy/issues/41> via
#48 <https://github.com/devops-coop/ansible-haproxy/pull/48>.

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/devops-coop/ansible-haproxy/issues/41#event-841036262>,
or mute the thread
